ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -246,8 +246,8 @@ func ResourceIBMContainerVpcCluster() *schema.Resource {
 				Optional:         true,
 				Default:          ingressReady,
 				DiffSuppressFunc: flex.ApplyOnce,
-				ValidateFunc:     validation.StringInSlice([]string{masterNodeReady, oneWorkerNodeReady, ingressReady}, true),
-				Description:      "wait_till can be configured for Master Ready, One worker Ready or Ingress Ready",
+				ValidateFunc:     validation.StringInSlice([]string{masterNodeReady, oneWorkerNodeReady, ingressReady, normal}, true),
+				Description:      "wait_till can be configured for Master Ready, One worker Ready or Ingress Ready or normal",
```

Comment:
wait_till can be configured for Master Ready, One Worker Ready, Ingress Ready or Normal